### ISSUE TITLE
Change latex strings to raw strings

### ISSUE THREAD

**Issue by jaladh-singhal** [tardis-sn/tardis#1206]
Throughout the tardis codebase, we have multiple LATEX strings which are written as simple strings - they should be converted to raw strings so that we don't need to write double `\\`.

For e.g.:
https://github.com/tardis-sn/tardis/blob/e8ab6605d91b69d389a16802be8e06f8ef0cc51a/tardis/plasma/properties/general.py#L40-L42
```
latex_name = (r'g_{\textrm{electron}}',) 
latex_formula = (r'\Big(\dfrac{2\pi m_{e}/\beta_{\textrm{rad}}}{h^2}\Big)^{3/2}',) 
```
 ## Pointers 
Search such latex strings throughout the tardis codebase and fix them - they'll be mostly values assigned to `latex_name`, `latex_formula`, etc.

**Comment by Sanchayan-Bhowal** [comment]
Is anyone working on this issue else I would like to work on it

**Comment by jaladh-singhal** [comment]
@Sanchayan-Bhowal This has already been implemented by #1283 

I should close this for now.
